JenkinsSolutions/q8.txt

```diff
@@ -0,0 +1,17 @@
+import groovy.json.JsonSlurper
+node {
+   
+    stage('fetch') { 
+        URL url = new URL("https://jsonplaceholder.typicode.com/todos");
+       
+         def card = new JsonSlurper().parse(url)
+        //  println(card)
+        for(def list : card) {
```

Comment:
i'd suggest todo here instead of list, since it iterates to represent each todo.